lib/tracker.ex

```diff
@@ -19,6 +19,11 @@ defmodule Tracker do
     end)
   end
 
+  def complete? do
+    {total, completed} = Agent.get(__MODULE__, &(&1))
+    total == Enum.count(completed)
```

Comment:
I guess this is a question for @felipesere. The progress seems to work pretty well so I trust the tracker at this point :)